### ISSUE TITLE
Installing IntelliJ IDEA with what association for Android development?

### ISSUE THREAD

**Issue by tombirmingham** [flutter/flutter#10053]
I'm installing IntelliJ IDEA and it asks to Create associations: .java, .groovy, .kt.

I'm right now only doing Android development, so is it correct to select .java?

See attached install dialog pic.

![intellij-idea-1](https://cloud.githubusercontent.com/assets/250991/26016272/37ab7f0a-3719-11e7-9423-6fb512a63410.jpg)

> For more information about diagnosing and reporting Flutter bugs, please see [https://flutter.io/bug-reports/](https://flutter.io/bug-reports/).

**Comment by tombirmingham** [comment]
So I checked .java.  Added the jdk, flutter, and dart.  All seemed ok.

Created a flutter project and it has the androidmanifest.xml issues of not being resolved.  I was able to launch the app and debug main.dart.  So that is great.

So now not sure how to set breakpoints in the android java code to sending messages to android java from dart.

So it looks like the IntelliJ IDE knows about Flutter and Dart but not android java.  So...

I created the project using the cmd line, flutter create and imported into Android Studio with android.  All is ok for Studio and debugging android java.  But not main.dart.  So I launched IntelliJ and it has the androidmanifest error for android framework.  Left that alone.  I was able to debug main.dart with emulator.

So all seems ok to me except I can't from one IDE, whether Studio or IntelliJ debug both flutter dart and android java?  Anyhow, I included the project as attached.
[test_myapp.zip](https://github.com/flutter/flutter/files/998852/test_myapp.zip)
